sdk/ml/azure-ai-ml/tests/test_configs/pipeline_jobs/helloworld_pipeline_job_with_node_services_inline_job.yml

```diff
@@ -13,11 +13,11 @@ jobs:
     command: echo Hello World & sleep 1h
     environment: azureml:AzureML-sklearn-0.24-ubuntu18.04-py37-cpu:1
     services:
-      "my_jupyter":
-        job_service_type: "Jupyter" # Jupyter Notebook
+      "my_ssh":
+        job_service_type: "Jupyter"
```

Comment:
We should remove Jupyter type as it is not supported.
Would pipeline team needs any changes following this?